README.md

```diff
@@ -54,6 +54,39 @@ torchaudio.save('foo_save.mp3', sound, sample_rate) # saves tensor to file
 ```
 
 API Reference
------------
+-------------
 
 API Reference is located here: http://pytorch.org/audio/
+
+Conventions
+-----------
+
+Torchaudio is standardized around the following naming conventions.
+
+* waveform: a tensor of audio samples with shape (channels, time)
+* sample_rate: the rate of audio samples (samples per second)
+* specgram: a tensor of spectrogram with shape (channels, time)
+* mel_specgram: a mel spectrogram with shape (channels, time)
+* hop_length: the number of samples between the starts of consecutive frames
+* n_freqs: the number of bins in a linear spectrogram
+* min_freq: the lowest frequency of the lowest band in a spectrogram
+* max_freq: the highest frequency of the highest band in a spectrogram
+* n_fft: the number of fourier bins
+* n_mfcc, n_mels: to be consistent with other similarly named variables, with shape (channel, n_mfcc, time) and (channel, n_mels, times)
+* win_length: the length of the STFT window
+* window_fn: for functions that creates windows e.g. torch.hann_window
+
+Transforms expect the following shapes. In particular, the input of all transforms and functions assumes channel first.
+
+* Spectrogram: (channel, time) -> (channel, frequency, time, 2)
+* MelScale: (channel, time) -> (channel, n_mels, time)
+* MFCC: (channel, time) -> (channel, n_mfcc, time)
+* MuLawEncode: (channel, time) -> (channel, n_mulaw, time)
+* MuLawDecode: (channel, n_mulaw, time) -> (channel, time)
+* Resample: (channel, time) -> (channel, time)
+* STFT: (channel, time, 2) -> (channel, frequency, time, 2).
+* ISTFT: (channel, frequency, time, 2) -> (channel, time, 2).
+
+A spectrogram can be converted to DB scale or Mel scale, using AmplitudeToDB and AmplitudeToMel.
+
+The Kaldi compliance interface follows Kaldi's interface, and provides access to: Kaldi's `fbank`, `spectrogram`, and `resample_waveform`.
```

Comment:
Not sure if this is part of conventions. I would not include it as it is mentioned here https://github.com/pytorch/audio/pull/166/files#diff-04c6e90faac2675aa89e2176d2eec7d8R15

Reply:
I agree with you. The Kaldi interface is to help onboard, but not really part of the conventions.